test/foundry/BlastNativeYield.t.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {IOwnableTwoSteps} from "../../contracts/interfaces/IOwnableTwoSteps.sol";
+import {OwnableTwoSteps} from "../../contracts/OwnableTwoSteps.sol";
+import {BlastNativeYield} from "../../contracts/BlastNativeYield.sol";
+import {TestHelpers} from "./utils/TestHelpers.sol";
+import {YieldMode as IBlast__YieldMode, GasMode as IBlast__GasMode} from "../../contracts/interfaces/IBlast.sol";
+
+import {MockBlastPoints} from "../mock/MockBlastPoints.sol";
+import {MockBlastYield} from "../mock/MockBlastYield.sol";
+
+contract BlastNativeYieldOwnableTwoSteps is BlastNativeYield, OwnableTwoSteps {
```

Comment:
similar to the test file below, you also don't need this, don't blindly copy and paste boilerplate code just because another test needs it

Reply:
https://github.com/LooksRare/contracts-libs/pull/103/commits/666c866d0326865e1a1e8178aca6a9452a714be3